### ISSUE TITLE
Query plan visualisation tool

### ISSUE THREAD

**Issue by dementiacs** [Studio3T/robomongo#1333]
Would be great if a query plan visualisation tool could be built in in much the same way as Mongo Compass does.

**Comment by simsekgokhan** [comment]
Hi @dementiacs , thanks for the suggestion. Are you talking about this feature: https://www.mongodb.com/blog/post/visualizing-your-data-with-mongodb-compass or something else?

**Comment by dementiacs** [comment]
Yes, something like that would be useful in Robomongo. Would save me jumping into Compass, which I hate, as it is quite slow even if it is pretty. Pretty is fine but what wins day to day is functionality and features that means we only need a single tool for Mongo admin and development.
